openmdao/core/tests/test_impl_comp.py

```diff
@@ -531,10 +531,12 @@ def setUp(self):
         group.connect('comp1.a', 'sub.comp2.a')
         group.connect('comp1.b', 'sub.comp2.b')
         group.connect('comp1.c', 'sub.comp2.c')
+
         group.connect('comp1.a', 'sub.comp3.a')
         group.connect('comp1.b', 'sub.comp3.b')
         group.connect('comp1.c', 'sub.comp3.c')
 
+        global prob
```

Comment:
What's the reason for this? We definitely don't want these 3 statements in the docs.

Reply:
`global` statements do not appear in the docs.. not sure what 3 statements you are referring to

Reply:
I thought this was in a System setup method for some reason. I withdraw.